gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -42,51 +43,88 @@
 
     private static final int ENTITY_MESSAGE_TYPE_PARAMETER_INDEX = 1;
 
-    private final DatastoreWrapper datastore;
+    private DatastoreWrapper datastore;
     private final Options options;
+    private final boolean multitenant;
 
     /**
+     * Creates new instance of {@code DatastoreStorageFactory}.
+     * <p>Same as calling {@link #newInstance(Datastore, boolean)} with {@code false} second argument.
+     *
      * @param datastore the {@link Datastore} implementation to use.
      * @return creates new factory instance.
      * @see DatastoreOptions
      */
     @SuppressWarnings("WeakerAccess") // Part of API
     public static DatastoreStorageFactory newInstance(Datastore datastore) {
-        return new DatastoreStorageFactory(datastore);
+        return new DatastoreStorageFactory(datastore, false);
     }
 
-    /* package */ DatastoreStorageFactory(Datastore datastore) {
-        options = new Options();
-        this.datastore = new DatastoreWrapper(datastore, options);
+    /**
+     * Creates new instance of {@code DatastoreStorageFactory}.
+     *
+     * @param datastore   the {@link Datastore} implementation to use.
+     * @param multitenant shows if storage factory is configured to serve a multitenant application
+     * @return creates new factory instance.
```

Comment:
By the current javadoc I will expect that this is void method.
Better something like : 
`@return new instance if the {@code  DatastoreStorageFactory}`